src/api.ts

```diff
@@ -58,6 +58,10 @@ class Radar {
       throw new RadarPublishableKeyError('Secret keys are not allowed. Please use your Radar publishable key.');
     }
 
+    if (Config.isInitialized()) {
+      Logger.error('Radar.initialize() called more than once.');
```

Comment:
Given that we're handling this, should we `Logger.warn` instead?

Reply:
I agree it's a "warn" and not an error, but I also sort of want this to fire on a production site. Maybe I will just change to `console.warn` instead.